spec/active_graph/base_spec.rb

```diff
@@ -135,47 +135,47 @@ def get_object_by_id(id)
 
     describe 'after_commit hook' do
       it 'gets called when the root transaction is closed' do
-        data = false
+        data = 0
         expect do
           subject.transaction do |tx1|
             subject.transaction do |tx2|
               subject.transaction do |tx3|
-                tx3.after_commit { data = true }
+                tx3.after_commit { data += 1 }
               end
             end
           end
-        end.to change { data }.to(true)
-        expect(data).to be_truthy
+        end.to change { data }.to(1)
```

Comment:
This error was not caught before because to the test only expected `data` to equal `true`, rather than checking the number of times it was called.